pom.xml

```diff
@@ -56,13 +50,32 @@
     </repositories>
 
     <dependencies>
-        <!-- Minecraft plugin -->
+        <!-- Paper Dependency -->
         <dependency>
-            <groupId>org.spigotmc</groupId>
-            <artifactId>spigot-api</artifactId>
-            <version>1.16.5-R0.1-SNAPSHOT</version>
+            <groupId>io.papermc.paper</groupId>
+            <artifactId>paper-api</artifactId>
+            <version>1.20.1-R0.1-SNAPSHOT</version>
```

Comment:
I think we can go ahead and merge this in, if you change this to `1.18.2-R0.1-SNAPSHOT` and the mockbukkit dep to:

```
        <dependency>
            <groupId>com.github.seeseemelk</groupId>
            <artifactId>MockBukkit-v1.18</artifactId>
            <version>2.85.2</version>
            <scope>test</scope>
        </dependency>
```